src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/custom.py

```diff
@@ -102,32 +104,39 @@ def _validate_start_time(start_time, end_time):
     return result_time
 
 
-# pylint: disable=too-many-arguments
-def list_activity_logs(client, correlation_id=None, resource_group=None, resource_id=None,
-                       resource_provider=None, start_time=None, end_time=None,
+# pylint: disable=too-many-arguments, line-too-long
```

Comment:
I saw this pattern been adopted in multiple places. Why not extract it into a method? like
``` Python
def single(collection):
    return len([x for x in collection if x]) > 1
```